docs/permissions/getting-started.md

```diff
@@ -20,20 +20,12 @@ The permissions framework depends on a few other Backstage systems, which must b
 
 The permissions framework itself is new to Backstage and still evolving quickly. To ensure your version of Backstage has all the latest permission-related functionality, it’s important to upgrade to the latest version. The [Backstage upgrade helper](https://backstage.github.io/upgrade-helper/) is a great tool to help ensure that you’ve made all the necessary changes during the upgrade!
 
-### Enable service-to-service authentication
-
-Service-to-service authentication allows Backstage backend code to verify that a given request originates from elsewhere in the Backstage backend. This is useful for tasks like collation of catalog entities in the search index. This type of request shouldn’t be permissioned, so it’s important to configure this feature before trying to use the permissions framework.
-
-To set up service-to-service authentication, follow the [service-to-service authentication docs](../auth/service-to-service-auth.md).
-
```

Comment:
IIRC We don't need this anymore right?

Reply:
Correct, in the new system it's always set up correctly.